build/ci.go

```diff
@@ -347,6 +347,7 @@ func doFmt(cmdline []string) {
 		"--tests",
 		"--disable-all",
 		"--enable=gofmt",
+		"--timeout=2m",
```

Comment:
Is this related to this PR?

Reply:
Without this, the linter is failed with timeout error.